.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -23,5 +23,16 @@ A clear and concise description of what you expected to happen.
 ## Actual Results (including error logs, if applicable)
 A clear and concise description of what actually did happen.
 
+## Reproducible
+   * [ ] Always 
+   * [ ] Sometimes
+   * [ ] Non-Reproducible
+   
+## Version/Tag number
+A clear input of your version/Tag number
+
+## Environment setup
+A clear input of your environment may help us to reproduce the issue.  
```

Comment:
I’m not sure I’m clear on what you mean by “environment setup” - can you clarify?

Reply:
for example:
In secrets provider it may be a different case if we run on oc3.10 or gke
In appliance running with cluster ? master only ? with LB ?  on which VM ? every input that may help us to reproduce it.

Reply:
thanks! I might try to wordsmith it a little to give people some guidance about what kind of info (like this) we're looking for